### PR TITLE
Add docsify-pangu and Update weex-ui

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ jobs:
             fetch-depth: 0
         - name: "lint: awesome-lint"
           uses: jthegedus/github-action-awesome-lint@v0.1.0 # https://github.com/jthegedus is part of Docsify core team, so we will use Tag versioning over commit_sha
+          env:
+            AWESOME_LINT_VERSION: "0.14.0" #0.15.0 Invalid URL: http:#showcase
```
Comment (sy-records) [inline .github/workflows/lint.yml:16]: @jthegedus hi, Do you have time to review this PR?

Comment (jthegedus) [reply .github/workflows/lint.yml:16]: Can you elaborate? What is the invalid URL?

Comment (sy-records) [reply .github/workflows/lint.yml:16]: https://github.com/docsifyjs/awesome-docsify#contents These links are parsed incorrectly.

Comment (sy-records) [reply .github/workflows/lint.yml:16]: Not a complete link, but an anchor

Comment (jthegedus) [reply .github/workflows/lint.yml:16]: Okay, do we need the anchor? Can you raise an issue with the root tool https://github.com/sindresorhus/awesome-lint

Comment (jthegedus) [reply .github/workflows/lint.yml:16]: I didn't see this issue in a prev PR I just recently merged

Comment (jthegedus) [reply .github/workflows/lint.yml:16]: I think it was a weird runtime error as I reran your other PR's CI check and it didn't fail on that issue (it failed on other issues I have resolved in master so a rebase should fix)

Comment (jthegedus) [reply .github/workflows/lint.yml:16]: I think this should be reverted as I would like to keep up to date with the `latest` of `awesome-lint`

Comment (sy-records) [reply .github/workflows/lint.yml]: done.

**Comment by jthegedus** [comment]
@sy-records here's the issue in the original tool https://github.com/sindresorhus/awesome-lint/issues/114

**Pull request merged**